src/som/interpreter/actors/Actor.java

```diff
@@ -64,9 +66,15 @@ public static void traceActorsExceptMainOne(final SFarReference actorFarRef) {
   /** Is scheduled on the pool, and executes messages to this actor. */
   private final ExecAllMessages executor;
 
+  /**
+   * Contains the operations for the management of messages between actors.
+   */
+  private final LocalManager localManager;
```

Comment:
This is something I don't want to integrate like that, because it makes creating actors more expensive. The local manager is a huge object. It has an object buffer, two, hash maps, and other state.

I think, we might want to put this into a subclass of actor that is only used when executing with a debugger.

Or, we change the whole approach in a way that the actors themselves do not have or do not need a local manager.